<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emulationGUIClass</class>
 <widget class="QMainWindow" name="emulationGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>emulationGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="stepbystepButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Step by Step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="controlButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>CPU</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>141</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Stack</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SP:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="spValue">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>19</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Status Flag</string>
     </property>
     <widget class="QCheckBox" name="nbit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="vbit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>O</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ubit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>U</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="bbit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ibit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="zbit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="dbit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Registros</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pcValue">
      <property name="geometry">
       <rect>
        <x>192</x>
        <y>19</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="aValue">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>19</y>
        <width>30</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>116</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>A:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="yValue">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>19</y>
        <width>30</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xValue">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>19</y>
        <width>30</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>172</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>PC:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Contadores</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ciclos:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cpucycleValue">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>29</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="nextcpucycle">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Cycle</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>321</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Vista del contexto</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>281</width>
       <height>192</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>24</number>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="emulationGUI.qrc"/>
 </resources>
 <connections/>
</ui>
